packages/autorest.python/autorest/m4reformatter/__init__.py

```diff
@@ -665,6 +671,9 @@ def _update_body_parameter_helper(
                 body_param["clientDefaultValue"] = body_type["value"]
         body_param["flattened"] = flattened
         body_param["isPartialBody"] = is_partial_body
+        body_param["restApiName"] = body_param["restApiName"] or to_lower_camel_case(
+            body_param["clientName"]
+        )
```

Comment:
modelerfour does not output `serializedName` but it is needed when generating sample so we have to convert it by ourselves:
![image](https://user-images.githubusercontent.com/70930885/176406771-185e1d15-787b-4fff-8a8a-f4a73d678fee.png)